README.md

```diff
@@ -17,7 +17,7 @@
 NNI (Neural Network Intelligence) is a toolkit to help users run automated machine learning (AutoML) experiments.
 The tool dispatches and runs trial jobs generated by tuning algorithms to search the best neural architecture and/or hyper-parameters in different environments like local machine, remote servers and cloud.
 
-### **NNI [v0.9](https://github.com/Microsoft/nni/releases) has been released!**
+### **NNI [v0.9](https://github.com/Microsoft/nni/releases) has been released! &nbsp;<img width="48" src="docs/img/release_icon.png">**
 <p align="center">
   <a href="#nni-v05-has-been-released"><img src="docs/img/overview.svg" /></a>
```

Comment:
I have little confuse why there "v05" is here...." <a href="#nni-v05-has-been-released"><img src="docs/img/overview.svg" /></a>"

Reply:
It's an invalid router to avoid the picture linked other page. I have changed V05 to V09.